tests/admin_custom_urls/tests.py

```diff
@@ -17,7 +20,24 @@ class AdminCustomUrlsTest(TestCase):
     * The ModelAdmin for Action customizes the add_view URL, it's
       '<app name>/<model name>/!add/'
     """
-    fixtures = ['users.json', 'actions.json']
+
+    @classmethod
+    def setUpTestData(cls):
+        User.objects.create(
+            pk=100, username='super', first_name='Super', last_name='User', email='super@example.com',
+            password='sha1$995a3$6011485ea3834267d719b4c801409b8b1ddd0158', is_active=True, is_superuser=True,
+            is_staff=True, last_login=datetime.datetime(2007, 5, 30, 13, 20, 10),
+            date_joined=datetime.datetime(2007, 5, 30, 13, 20, 10))
```

Comment:
Hmm no good reason, no. It was just how the mechanical conversion turned out. Prints on one line, hit enter at commas near the end of 120 char column, done. It's also relatively consistent with how the tests are formatted, but that's neither here nor there, and wasn't a motivation.

I can change if you think it needs it though.